utils/rpms/packaging/Dockerfile.mockbuild

```
@@ -29,6 +29,21 @@ RUN dnf -y install mock make                                        \
                    python-srpm-macros rpmdevtools &&                \
     dnf -y clean all
 
+# Install bullseye coverage
+ARG JENKINS_URL
+ARG BULLSEYE_KEY
+RUN if [ "x$BULLSEYE" != "x" ]; then                                                \
```

```suggestion
RUN set -x; if [ "x$BULLSEYE" != "x" ]; then                                                \
```

temporarily, just to confirm what this step is actually doing.

---

But per my other comment here, I suspect this addition is not actually having any positive effect on this effort and can be removed.  It's the bullseye RPM that is doing the heavy lifting here.